backend/__tests__/event.js

```diff
@@ -7,7 +7,10 @@ describe("event file", () => {
     return {
       description: "Description",
       event: 'event_slug',
-      storyline: 'storyline_slug'
+      storyline: 'storyline_slug',
```

Comment:
Let's not set the default values as we want to make sure they are added by default